src/main/scala/tech/cryptonomic/conseil/util/Conversion.scala

```diff
@@ -0,0 +1,27 @@
+package tech.cryptonomic.conseil.util
+
+trait Conversion[F[_], FROM, TO] {
+
+  def convert(from: FROM): F[TO]
+
+}
+
+object Conversion {
```

Comment:
Reminder to document this file when you get to it.

Reply:
Sure, I missed some doc here and there and will cover everything before the final merge

Reply:
done